examples/connext_dds/CMakeLists.txt

```diff
@@ -64,7 +67,9 @@ add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/batching/c++")
 add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/batching/c++03")
 
 # build_systems
-add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/build_systems/cmake")
+if(${RTICONNEXTDDS_VERSION} VERSION_GREATER_EQUAL ${REQUIRED_VERSION})
```

Comment:
Add an `else` statement with a message saying something like "skipping example <name> because it requires ConnextDDS version <required version>". Do this for the other examples too

Reply:
Should we do hat in the example itself to make it cleaner? Is that possible?